CMakeLists.txt

```diff
@@ -283,6 +295,7 @@ endif()
 if(CDEPS)
   add_dependencies(ufs cdeps::cdeps)
   list(APPEND _ufs_defs_private FRONT_CDEPS_DATM=cdeps_datm_comp)
+  list(APPEND _ufs_defs_private FRONT_CDEPS_DOCN=cdeps_docn_comp)
```

Comment:
Is this going to work when `APP=NG-GODAS`? I am speculating since `NG-GODAS` has an active ocean.

Reply:
Since we change the CDEPS module names under UFS model, this will work without any issue and there won't be any conflict. BTW, all RTs including NG-GODAS one passing at this point.

Reply:
Thanks for clarifying.